kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java

```diff
@@ -314,7 +314,7 @@ public void process(BeanDefinition<?> beanDefinition, ExecutableMethod<?, ?> met
                 .orElseGet(() -> applicationConfiguration.getName().map(s -> s + '-' + NameUtils.hyphenate(beanType.getSimpleName())).orElse(null));
         final OffsetStrategy offsetStrategy = consumerAnnotation.enumValue("offsetStrategy", OffsetStrategy.class)
                 .orElse(OffsetStrategy.AUTO);
-        final AbstractKafkaConsumerConfiguration<?, ?> consumerConfigurationDefaults = beanContext.findBean(AbstractKafkaConsumerConfiguration.class, Qualifiers.byName(groupId))
+        final AbstractKafkaConsumerConfiguration<?, ?> consumerConfigurationDefaults = getConsumerConfigurationBean(groupId)
                 .orElse(defaultConsumerConfiguration);
```

Comment:
I think I'd prefer to see the `.orElse` logic encapsulated by `getConsumerConfigurationBean` instead of it returning an `Optional`. Perhaps rename that method to `getConsumerConfigurationDefaults`.